src/coreclr/src/pal/src/thread/context.cpp

```diff
@@ -428,7 +430,7 @@ void CONTEXTToNativeContext(CONST CONTEXT *lpContext, native_context_t *native)
     }
 #undef ASSIGN_REG
 
-#if HAVE_GREGSET_T || HAVE_GREGSET_T
+#if (HAVE_GREGSET_T || HAVE_GREGSET_T) && !HAVE_FPREGS_WITH_CW
```

Comment:
A nit - could you please create a separate #if !HAVE_FPREGS_WITH_CW / #endif around the whole preexisting #if / #endif block instead? It seems it would express the intent in an easier to see way. The same below.

Reply:
Moved them on separate line and annotated the `endif`s for readability.